crates/ide_completion/src/completions/attribute.rs

```diff
@@ -36,8 +38,13 @@ pub(crate) fn complete_attribute(acc: &mut Completions, ctx: &CompletionContext)
             "feature" => lint::complete_lint(acc, ctx, &parse_tt_as_comma_sep_paths(tt)?, FEATURES),
             "allow" | "warn" | "deny" | "forbid" => {
                 let existing_lints = parse_tt_as_comma_sep_paths(tt)?;
+
+                let clippy_lint_groups: Vec<Lint> =
+                    CLIPPY_LINT_GROUPS.iter().map(|g| g.lint.clone()).collect();
```

Comment:
Is there a way to avoid `clone`?

Reply:
The clone here should be basically free so that's not a problem.

Reply:
Also for completions sake it would be good to chain `DEFAULT_LINT_GROUPS` and `RUSTDOC_LINT_GROUPS` into this as well. Those 3 should be all groups we have. (You can collect them all into one vec here)